daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/Ast.scala

```diff
@@ -1202,16 +1202,10 @@ object Ast {
       languageVersion: LanguageVersion,
       metadata: PackageMetadata,
   ) {
-    import Ordering.Implicits._
-
     // package Name if the package support upgrade
     // TODO: https://github.com/digital-asset/daml/issues/17965
     //  drop that in daml-3
-    private[lf] val name: Option[Ref.PackageName] = {
-      if (languageVersion >= LanguageVersion.Features.packageUpgrades)
-        Some(metadata.name)
-      else None
-    }
+    private[lf] val name: Option[Ref.PackageName] = Some(metadata.name)
```

Comment:
@tudor-da this returns an Option because it is then assigned to an `Option[PackageName]` in `Package`. I didn't go through the rabbit hole of making it non-optional because I seem to remember you told me you would be working on that. I'm happy to add a TODO pointing to an issue if you have one. If I got it wrong and you're not working on that then please let me know, and I'll add a TODO for myself.

Reply:
Thanks!